test/runtests.jl

```diff
@@ -53,8 +53,8 @@ cd(dirname(@__FILE__)) do
                             p = addprocs(1; exeflags=`--check-bounds=yes --depwarn=error`)[1]
                             remotecall_fetch(()->include("testdefs.jl"), p)
                         else
-                            # single process testing, bail if mem limit reached, or, on an exception.
-                            isa(resp, Exception) ? rethrow(resp) : error("Halting tests. Memory limit reached : $resp > $max_worker_rss")
+                            # single process testing, bail if mem limit reached.
+                            isa(resp, Exception) || error("Halting tests. Memory limit reached : $resp > $max_worker_rss")
```

Comment:
To limit the number of places in the code where the printing of the seed has to be handled. As far as I could understand, the logic is not really modified here.